harness/adapters/google.py

```
@@ -0,0 +1,253 @@
+
+SYS_PROMPT = (
+    "You are an expert analog/mixed-signal IC designer. "
+    "Follow the user's Required sections exactly and return markdown only."
+    "NEVER use LaTeX or MathJax in your responses."
+)
+
+
+class GoogleAdapter(BaseAdapter):
+    name = "google"
+
+    def __init__(self, model: str | None = None, temperature: float = 0.2, max_tokens: int = 800):
```

Ruff (0.14.3)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)